CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 - Sample QC results moved to below summary 
 - Split some `Case Overview` section into a new `Patient and Physician` section
 - Removed tracking of patient's genetic sex
+- GCGI-943: Overrode HGVSp for BRAF V640E to be represented as V600E
```

Comment:
also need to add Alex's change of the maf file

Reply:
Ooh yes, done!